.github/workflows/update_release_refs.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+
+import yaml
+
+folder_path = sys.argv[1]
+# Define the folder where YAML files are located
+# folder_path = f"/manifests/{new_version}"  # Replace with the actual folder path
+
+# Get the folder name and use it as the new version
+new_version = os.path.basename(folder_path)
+
+# Define a list of component names that should have the same version as 'OpenSearch' and 'common-utils'
+same_version_components = ['OpenSearch', 'common-utils']
```

Comment:
Only OpenSearch/OpenSearch-Dashboards/functionaltestrepo has same 3 digit version number, others will have 4 digit numbers with 0 as the 4th digit.